packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -131,7 +131,7 @@ const BlockComponent = forwardRef(
 
 		useEffect( () => {
 			if ( ! isMultiSelecting && ! isNavigationMode && isSelected ) {
-				focusTabbable( ! isMounting.current );
+				focusTabbable( true );
```

Comment:
Not sure if it's still needed or not but the main reason for this was that we wanted to focus the first paragraph of the columns block on insert. Since then the columns block has been refactored to avoid any paragraph on insert. So it might not be needed anymore but we still need to confirm with other blocks with inner blocks (Media & Text, third party blocks...)

Reply:
Thanks for the comment! I was wondering what this is used for, so I wanted to see which e2e tests fail. :) So that's good to know that we could remove this behaviour. Changing focus behaviour between mount and further re-render seems a bit fragile.